### PR TITLE
Schemas for query contracts

### PR DESCRIPTION
**Pull request by NinoLipartiia**

### CODE REVIEW

File: scripts/schema.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+cargo run --example schema > /dev/null
+
+# When all schemas are ready, can create schemas like this:
+# for f in ./contracts/*
+# do
+#   cd "$f"
+#   CMD="cargo run --example schema"
+#   cargo run --example schema
+#   cd "$START_DIR"
+# done
```
Comment (Buckram123) [inline scripts/schema.sh:22]: I'm ok with errors during schema generation for now, what do you think? @deveusss @mikedotexe @TrevorJTClarke

**Review by Buckram123 — APPROVED** [review]
LGTM

File: scripts/schema.sh (hunk 1 above)
Comment (mikedotexe) [reply scripts/schema.sh:22]: yeah, for now it's fine, we mainly need the contracts to be ready for the audit. I also trust the schema issues will be much easier once we merge in the separate contracts that are being split out

**Review by mikedotexe — APPROVED** [review]
Thanks!

**Pull request merged**